numbers.js

```diff
@@ -0,0 +1,59 @@
+console.log('num1+num2 represents the equation to find the sum of these numbers');
```

Comment:
Do the tasks one at a time.

1. Can you assign a number to a variable named `num1`?
1. Can you assign a number to a variable named `num2`?
1. Can you use `+` with `num1` and `num2`? What does it represent?

For each task, both explain in words what the code is doing and do the thing described in the task.

I gave an example in the README: https://github.com/jfarmer/javascript-milestones#demonstrations